petri/src/vm/runtime.rs

```diff
@@ -161,14 +156,6 @@ impl PetriVm {
         /// Useful if you've rebooted the vm or are otherwise expecting a fresh connection.
         pub async fn wait_for_agent(&mut self) -> anyhow::Result<PipetteClient>
     );
-    petri_vm_fn!(
-        /// Wait for VTL 2 to report that it is ready to respond to commands.
-        /// Will fail if the VM is not running OpenHCL.
-        ///
-        /// This should only be necessary if you're doing something manual. All
-        /// Petri-provided methods will wait for VTL 2 to be ready automatically.
-        pub async fn wait_for_vtl2_ready(&mut self) -> anyhow::Result<()>
```

Comment:
I think this is being used in internal tests

Reply:
bluh

Reply:
Maybe such tests should use the new interface rather than go through the GED. When petri supports the Hyper-V backend, we won't have access to the GED... (OTOH, Hyper-V won't even consider the VM started until the GED connects, so I don't know.)

Reply:
OK, I've restored this but switched it to use the new diag_client functionality for waiting for readiness.